internal/service/atlasuser/data_source_atlas_users_test.go

```diff
@@ -200,16 +201,17 @@ func TestAccConfigDSAtlasUsers_InvalidAttrCombinations(t *testing.T) {
 	}
 }
 
-func fetchOrgUsers(t *testing.T, orgID string) *admin.PaginatedAppUser {
+func fetchOrgUsers(t *testing.T, orgID string) *admin20240805.PaginatedAppUser {
 	t.Helper()
-	users, _, err := acc.ConnV2().OrganizationsApi.ListOrganizationUsers(context.Background(), orgID).Execute()
+	connV220240805 := acc.MongoDBClient.AtlasV220240805
```

Comment:
can we define acc.ConnV220240805() within `factory.go` for consistency? We can also leave a comment in this function that it is temporary and will be removed once use of preview SDK is replaced.

Reply:
Created [this ticket](https://jira.mongodb.org/browse/CLOUDP-281945) to cover this suggested change 👍